src/message/customer-information/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useRef } from 'react';
+import { Form } from 'antd';
+import { GenericForm } from '../../components/form';
+import { triggerMessageAndHandleResponse } from '../util';
+import {
+  GetCustomerProps,
+  GetCustomersRequest,
+  CustomerPayload,
+} from '../../model/CustomerInformation';
+
+export const GetCustomers: React.FC<GetCustomerProps> = ({ station }) => {
```

Comment:
very minor, is there a reason this component is named `GetCustomers` rather than `CustomerInformation` ? since the main request is known as the `CustomerInformationRequest`, would it make sense to name this component `CustomerInformation` instead? 🤔